corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -1,6 +1,8 @@
 /*global DOMPurify, Marionette */
 
-hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
+hqDefine("cloudcare/js/formplayer/menus/views/query", [
+    'hqwebapp/js/hq.helpers',
```

Comment:
Just noticed the test failure - cloudcare doesn't use requirejs yet, so you can't include dependencies here. There are some notes on this [here](https://github.com/dimagi/commcare-hq/blob/master/docs/js-guide/migrating.md#note-on-editing-unmigrated-javascript).

Reply:
Thanks Jenny.

It looks like `hq.helpers` doesn't define `hqHelp` in the same way other hqModules except. It returns a `1` instead of the functions. Is it possible to import these using `hqImport`. Or should I move `hqHelp` function elsewhere.

Reply:
Ah, I wrote that without noticing it was `hq.helpers`. You don't need to import it at all. For non-requirejs pages, it gets included [here](https://github.com/dimagi/commcare-hq/blob/78a848bfad7765634835ed7201e149cf9c997d58/corehq/apps/hqwebapp/templates/hqwebapp/includes/core_libraries.html#L120).

It'd be angelic to add a comment noting this file depends on `hq.helpers`, or to just comment out this line instead of removing it. When web apps is migrated to requirejs, it will need to be added just as you've done. These kinds of dependencies, where the module just sets up jquery plugins instead of returning functions, are easy to miss when doing a migration so they cause bugs often.

Reply:
Great, I have added the comment.